packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -117,6 +159,9 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			__experimentalCanUserUseUnfilteredHTML: canUseUnfilteredHTML,
 			__experimentalUndo: undo,
 			outlineMode: hasTemplate,
+			__experimentalCreateEntity: createEntity,
+			__experimentalUserCanCreate:
```

Comment:
Is it appropriate to expose this property? Would `__experimentalUserCanCreateEntities` be a better name?